simphony/testing/abc_check_mesh.py

```diff
@@ -384,11 +400,14 @@ def test_update_item_points(self):
         container = self.container
         uids = self._add_items(container)
         item = self.get_operation(container, uids[2])
+        point_uids = [
+            container.add_point(Point((1.0 * i, 1.0 * i, 1.0 * i)))
```

Comment:
I just took a quick look at this PR and think I might be lost on its intent.  I see here that the points are added to the container here for ```test_update_item_points```,  but from the PR description I would have thought this would occur somewhere else (like MeshElementOperationsCheck's setup) so that other tests related to edges, cells and faces would be using a container that had points.  

besides this, lgtm